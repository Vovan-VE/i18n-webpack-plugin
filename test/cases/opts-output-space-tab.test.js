import fs from 'fs';
import processFile from '../cases.setup';

describe('opts-output-space-tab', () => {
  let extracted;
  const outputSpace = '\t';

  beforeAll(() => processFile('simple.code.js', { outputSpace })
    .then(({ file }) => {
      extracted = fs.readFileSync(file, 'utf8');
    }));

  it('should return extracted keys', () => {
    expect(extracted).toMatchSnapshot('opts-output-space-tab.test.js');
  });
});
