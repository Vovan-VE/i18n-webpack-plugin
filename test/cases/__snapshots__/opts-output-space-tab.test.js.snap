// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`opts-output-space-tab.test.js 1`] = `
"{
	\\"category1\\": {
		\\"Message 1\\": \\"\\",
		\\"Message 2\\": \\"\\"
	},
	\\"category2\\": {
		\\"Message 3\\": \\"\\"
	}
}"
`;
